lib/src/models/movie.dart

```diff
@@ -0,0 +1,30 @@
+import 'movie_result.dart';
+
+class Movie {
```

Comment:
Why you add this logic here?

Reply:
Because I need it to be able to export to GridHomePage, because it comes with a variable of type Movie as a parameter